README.adoc

```diff
@@ -26,6 +26,12 @@ maintain.
 Note: Lightway Core may be referred to by the internal project name, libhelium,
 in the source code.
 
+== Development Environment
+
+This project uses [pre-commit](https://pre-commit.com/) to enforce code standards.
```

Comment:
Thanks @expressvpn-tom-l for adding documentation. Changes LGTM. 

One minor thing, if we need to enforce this `formatting` check, this has to be done as part of CI.

Precommit only helps with client (dev) side verification. If there is some new developer, who missed installing precommit hook in their repo, we may end up getting unformatted code which might be missed in code review.

Adding it to CI ensures, new code changes adhered to the formatting standards

Reply:
OK, I'll add the CI check in another PR.